default/python/universe_generation/universe_tables.py

```diff
@@ -141,6 +141,7 @@
 }
 
 # The maximum length of any starlane the galaxy generator can create (in uu).
+# MAX_STARLANE_LENGTH >= 2 * MIN_SYSTEM_SEPARATION
```

Comment:
Put this explanation to code. From my context it look like debug artefact, which may be possible issue.